src/components/views/auth/RegistrationForm.tsx

```diff
@@ -411,7 +411,7 @@ export default class RegistrationForm extends React.PureComponent<IProps, IState
     }
 
     private showPhoneNumber() {
-        const threePidLogin = !SdkConfig.get().disable_3pid_login;
+        const threePidLogin = !SdkConfig.get().loginScreen?.disable_3pid_login;
```

Comment:
Please revert this change and continue to use the element default config location for disable_3pid_login

Reply:
@jrschremmer I am not sure why to revert this change. there are multiple configurations inside the login screen which i have kept into the login screen config. it will be easier for a person to identify the configuration by that.
 "loginScreen": {
        "displaySecondaryLogo":false,
        "displaySecondaryVerticalLogo":false,        
        "displayLanguageSelector":false,    
        "disable_3pid_login": false,            
        "showHomeServerInfo": false,
        "showMatrixDefaultFooterLinks": false,
        "allowPhoneBasedAuthentication": false,
        "allowEmailBasedAuthentication": false,
        "changeSigninToLoginTextLabel": true,
        "backgroundColor": "(0,38, 119)",
        "logo":{
                "img":{
                "width": "150px",
                "height": "100px"
            }
        }

Reply:
done